### ISSUE TITLE
Add <optgroup> with labels to separate "priorityOptions" and the rest

### ISSUE THREAD

**Issue by danishmuneer** [country-regions/react-country-region-selector#104]
This is a feature request. 

It would be better to have the options wrapped in separate "optgroup" if "priorityOptions" is passed an array. Also the ability to set optgroup label to distinguish the two lists. I have attached a screenshot to explain the idea better. 

![Capture](https://user-images.githubusercontent.com/26395892/99914501-d779d080-2ccb-11eb-8422-6ffc72137add.PNG)

**Comment by benkeen** [comment]
Agreed, this would be nice. I'll see if I can get it into 4.0.0. Will tentatively shoot for that.
